exercises/hangman/README.md

```diff
@@ -0,0 +1,52 @@
+# Hangman
+
+Implement the logic of the hangman game using functional reactive programming.
+
+[Hangman][] is a simple word guessing game.
+
+[Hangman]: https://en.wikipedia.org/wiki/Hangman_%28game%29
+
```

Comment:
Since this file is supposed to be generated with `configlet generate`, I'm not comfortable with manually editing it, even to remove inaccurate (to this track) data.

Here are the options I can think of:
- Add a note to this exercise's `.meta/HINTS.md` that tells users to ignore the part about a 3rd-party framework.
- Leave inaccurate data as-is (least preferable)
- Submit a PR to exercism/problem-specifications to remove the reference to a 3rd-party framework, leaving it to individual tracks to add that information (i.e. [C# track does this already](https://github.com/exercism/csharp/blob/3f7ce0626176da8130781dcf8c6484968df8c69f/exercises/hangman/.meta/hints.md))
- Mark this exercise as `foregone` since there is no builtin FRP framework. There is a precedent for this with the `lens-person` exercise.
- *\[Insert other options here]*

Reply:
Thank you for the feecback, I have added a hint about that.